packages/storybook-builder-vite/codegen-importfn-script.js

```diff
@@ -16,7 +16,15 @@ const { normalizePath } = require('vite');
  */
 async function toImportFn(stories) {
     const objectEntries = stories
-        .map(file => `  './${normalizePath(path.relative(process.cwd(),file))}': async () => import('/@fs/${file}')`);
+        .map(file => {
```

Comment:
Wondering if we should do something like https://github.com/storybookjs/storybook/blob/d42f89a9d43cb7753e5484c9dd2bcf8677053239/lib/core-server/src/utils/watch-story-specifiers.ts#L21-L23

Having identical logic to Storybook is more favourable in my opinion. Something like:

```js
// Paths get passed either with no leading './' - e.g. `src/Foo.stories.js`,
// or with a leading `../` (etc), e.g. `../src/Foo.stories.js`.
// We want to deal in importPaths relative to the working dir, so we normalize
// Taken from https://github.com/storybookjs/storybook/blob/d42f89a9d43cb7753e5484c9dd2bcf8677053239/lib/core-server/src/utils/watch-story-specifiers.ts#L21-L23
function toImportPath(relativePath) {
    return relativePath.startsWith('.') ? relativePath : `./${relativePath}`;
}

...

async function toImportFn(stories) {
    const objectEntries = stories
        .map(file => `  './${toImportPath(normalizePath(path.relative(process.cwd(),file)))}': async () => import('/@fs/${file}')`);


```